src/chainparams.cpp

```diff
@@ -274,15 +274,15 @@ class SigNetParams : public CChainParams {
             vSeeds.emplace_back("2a01:7c8:d005:390::5");
             vSeeds.emplace_back("ntv3mtqw5wt63red.onion:38333");
 
-            consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000000000000000000d145533ce");
-            consensus.defaultAssumeValid = uint256S("0x00000128807d9175c494e24d805fc7854f7d79aa965cbb128342ad8b70cecfa5"); // 5348
+            consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000000000000019fd16269a");
+            consensus.defaultAssumeValid = uint256S("0x0000002a1de0f46379358c1fd09906f7ac59adf3712323ed90eb59e4c183c020"); // 9434
             m_assumed_blockchain_size = 1;
             m_assumed_chain_state_size = 0;
             chainTxData = ChainTxData{
                 // Data from RPC: getchaintxstats 4096 00000128807d9175c494e24d805fc7854f7d79aa965cbb128342ad8b70cecfa5
-                /* nTime    */ 1601382000,
-                /* nTxCount */ 5435,
-                /* dTxRate  */ 0.001898346323372538,
+                /* nTime    */ 1603986000,
+                /* nTxCount */ 9582,
+                /* dTxRate  */ 0.00159272030651341,
             };
```

Comment:
I have different Signet chaintxdata:
```
$ ./src/bitcoin-cli -signet getchaintxstats 4096 00000128807d9175c494e24d805fc7854f7d79aa965cbb128342ad8b70cecfa5
{
  "time": 1601382000,
  "txcount": 5435,
  "window_final_block_hash": "00000128807d9175c494e24d805fc7854f7d79aa965cbb128342ad8b70cecfa5",
  "window_final_block_height": 5348,
  "window_block_count": 4096,
  "window_tx_count": 4182,
  "window_interval": 2202970,
  "txrate": 0.001898346323372538
}
```

Reply:
Maybe I'm on a previous version of signet?

Reply:
Seeing the same output on a fresh new synced signet chain.

Reply:
The comment has not been updated https://github.com/bitcoin/bitcoin/pull/20263/files#r515620099 . Nice find.